src/language/ast.ts

```diff
@@ -288,7 +292,13 @@ export const QueryDocumentKeys: {
   UnionTypeExtension: ['name', 'directives', 'types'],
   EnumTypeExtension: ['name', 'directives', 'values'],
   InputObjectTypeExtension: ['name', 'directives', 'fields'],
-  SchemaCoordinate: ['name', 'memberName', 'argumentName'],
+
+  // Schema Coordinates
+  TypeCoordinate: ['name'],
+  MemberCoordinate: ['name', 'memberName'],
+  ArgumentCoordinate: ['name', 'memberName', 'argumentName'],
```

Comment:
Since this is always for fields at the moment, we could just use `fieldName`:

```suggestion
  ArgumentCoordinate: ['name', 'fieldName', 'argumentName'],
```